### PR TITLE
Create classifier service client

### PR DESCRIPTION
**Pull request by albatrosica**
Implements [[APPS-34](https://golemproject.atlassian.net/browse/APPS-34)] with subtasks: [[APPS-79](https://golemproject.atlassian.net/browse/APPS-79)],  [[APPS-81](https://golemproject.atlassian.net/browse/APPS-81)],  [[APPS-82](https://golemproject.atlassian.net/browse/APPS-82)]

Introduces client initial setup with backend integration. For setup instructions please see `readme` in the client dir.

In this version user can:
1. Type some text in textarea and submit it to the server.
2. With meta key down, highlight text and mark it (with single click). Text is then placed in the textarea for further edition/submission.
3. With meta and shift key down, user can reset selection.
4. After a submit, user can see results returned from the server and reset an app state.

Note: Below form area there is some html mocked for test purposes. See the arrow image. It has no text, so nothing is selected.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,23 @@ provider/output.txt
...
+
+# client production build
+client/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```
Comment (Wiezzel) [inline .gitignore]:
```suggestion
.env.*
```

File: client/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Client setup
```
Comment (Wiezzel) [inline client/README.md:1]: Please add system-level dependecies before first step (nodejs version, yarn version, etc.)

**Review by Wiezzel — COMMENTED** [review]
1. Yarn gives some warnings when building the project – not sure if this is something we do care about.
2. Why the emotion part of the result is not displayed?

**Comment by albatrosica** [comment]
> 1. Yarn gives some warnings when building the project – not sure if this is something we do care about.

No, these warnings comes from node modules dependencies - so no, we can do nothing about it.
> 2. Why the emotion part of the result is not displayed?

As for now, it always comes back with the same results, so I skipped it for now.

**Comment by Wiezzel** [comment]
@nataliakulas 

>As for now, it always comes back with the same results, so I skipped it for now.

Soon we will have a non-dummy backend implementation and we would like to test it end-to-end without the need to modify the frontend.

**Review by johny-b — COMMENTED** [review]
Looks fine to me, but I'm not fluent in frontend :(

File: client/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+| name | version | 
+|---|---|
+| node | 14.15.3 |
+| yarn | 1.22.4 |
+| npm | 6.14.9 |
+
+Note: default package manager for the project is `yarn` but you can use `npm` as well
+
+##DIY
```
Comment (pnowosie) [inline client/README.md]: Add space in-between `#` and title to make it markup header
```suggestion
## DIY
```

File: client/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Client setup
+
+##System requirements (at least)
```
Comment (pnowosie) [inline client/README.md]: Add space in-between # and title to make it markup header
```suggestion
## System requirements (at least)
```

File: client/src/App.js [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+  const handleReset = () => {
+    setText('');
+    setResult(undefined);
+  };
+
+  const handleFetch = (e) => {
+    e.preventDefault();
+
+    fetch('http://0.0.0.0:5000/api/classify', {
```
Comment (pnowosie) [inline client/src/App.js:22]: Please add "configuration variable" with `http://0.0.0.0:5000/api` and reuse to make a path here.
Not sure what is proper react way - global `const` somewhere at the top of the file is fine for me.

**Review by pnowosie — APPROVED** [review]
Excellent ❗ 
Just minors ahead

File: client/src/App.js (hunk 5 above)
Comment (albatrosica) [reply client/src/App.js:22]: Yes, it will be added later, probably with next PR.

**Pull request merged**